cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+import time
+from django.db import connections
+from django.db.utils import OperationalError
```

Comment:
Order and group your imports according to PEP-8 including separating them by blank lines